evap/staff/forms.py

```diff
@@ -183,6 +192,11 @@ def __init__(self, *args, **kwargs):
 
         self.fields['questionnaires'].queryset = Questionnaire.objects.filter(is_for_contributors=True).filter(
             Q(obsolete=False) | Q(contributions__course=self.course)).distinct()
+        
+        if not self.course.can_staff_edit:
+            # form is used as read-only course view
+            for field in self.fields.values():
+                field.disabled = True
```

Comment:
before i forget it again, this should go into a method `def disable_all_fields(form):`